### DISCUSSION TITLE
Update builtin policies through Artifactory

### DISCUSSION THREAD

**Discussion by danieljantosovic-ext54837** [aquasecurity/trivy#4651] [Q&A] [answered]
### Question

Hello, 
in our company we use Artifactory remote repositories as proxy to access registries on internet. 
I'm able to configure our artifactory as DB_REPOSITORY , DOWNLOAD_URL and DOWNLOAD_BASE_URL and i can successfully scan for example filesystem , but when I try scan configuration trivy starts downloading updates of policies through ghcr.io directly. 
Is there a way to redirect these updates to our artifactory too? 
Thank you.

### Target

Filesystem

### Scanner

Misconfiguration

### Output Format

Template

### Mode

Standalone

### Operating System

Ubunut 22.04

### Version

```bash
0.42.1
```

**Comment by simar7** [comment, marked as answer]
At the moment the misconfiguration policy bundle is hardcoded to download from `ghcr.io` https://github.com/aquasecurity/trivy/blob/dcc73e964a8a19e37fa495f6694be28a06ed3d64/pkg/policy/policy.go#L22

Having said that, we can look into adding this as a feature. I'll convert this discussion into a feature request.

**Reply by simar7** [reply]
Feature request opened here: https://github.com/aquasecurity/trivy/issues/4672

**Reply by danieljantosovic-ext54837** [reply]
Thank you.
